djangocms_picture/templates/djangocms_picture/picture.html

```diff
@@ -0,0 +1,77 @@
+ {% load thumbnail %}
+
+{% if instance.get_link %}
+    <a href="{{ instance.get_link }}"
+    {% if instance.link_target %} target="{{ instance.link_target }}"{% endif %}
+    {{ instance.link_attributes_str }}>
+{% endif %}
+
+{# start render figure/figcaption #}
+{% if instance.caption_text %}
+    <figure>
+{% endif %}
+{# end render figure/figcaption #}
+
+{% thumbnail instance.picture instance.get_size.size crop=instance.get_size.crop upscale=instance.get_size.upscale as thumbnail %}
+
+<img src="{% spaceless %}
+    {% if instance.external_picture %}
```

Comment:
Yeah I agree that the src output there is crazy, I'll read on how I might be able to pack that in a template tag.